SA/bibliography.bib

```diff
@@ -44,4 +44,11 @@ @online{electron-js
 	title = {Electron JS},
 	url = {https://www.electronjs.org},
 	urldate = {2023-09-28}
+}
+
+@online{reddit_visual_fp,
+	author = {Jameshfisher},
+	title = {A humble suggestion for visual pure functional programming},
+	url = {https://i.imgur.com/TOZZZ.png},
```

Comment:
Don't you want do set the reddit post? He did give more information in there by responding to questions